### PR TITLE
[ACA-2755] sso public url

### PR DESCRIPTION
**Pull request by mmuller88**
**Please check if the PR fulfills these requirements**
```
[x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-js-api/wiki/Commit-format)
[ ] Tests for the changes have been added (for bug fixes / features)
[ ] Docs have been added / updated (for bug fixes / features)
```

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
Allows to reuse the isPublicUrl from js-api inside of content-store

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/alfrescoApi.ts [hunk 1]
```diff
@@ -537,6 +537,10 @@ export class AlfrescoApi implements EventEmitter.Emitter {
         return this.config.authType === 'OAUTH';
     }
 
+    isPublicUrl(): boolean {
```
Comment (eromano) [inline src/alfrescoApi.ts]: Is better to call before (this.isOauthConfiguration())

File: src/alfrescoApi.ts [hunk 2]
```diff
@@ -537,6 +537,10 @@ export class AlfrescoApi implements EventEmitter.Emitter {
...
+         return this.oauth2Auth.isPublicUrl();
```
Comment (eromano) [inline src/alfrescoApi.ts]: Not sure why you need it can you do the same from outside?

Comment (mmuller88) [reply src/alfrescoApi.ts]: Good question. Regarding https://github.com/Alfresco/alfresco-ng2-components/pull/5363 I need to know for AuthGuardECM if that is a public url or not.

File: src/alfrescoApi.ts (hunk 1 above)
Comment (mmuller88) [reply src/alfrescoApi.ts]: Thx :)

Comment (eromano) [reply src/alfrescoApi.ts]: are you sure you need to check it? this.oauth2Auth &&

Comment (mmuller88) [reply src/alfrescoApi.ts]: It looks like oauth2Auth could be undefined if no parameter was provided in the constructor 
https://github.com/Alfresco/alfresco-js-api/blob/development/src/alfrescoApi.ts#L52

**Pull request merged**